scripts/install_dev_tools.sh

```diff
@@ -16,3 +16,8 @@ echo "poetry: $(poetry --version)"
 poetry install
 poetry lock --check
 npm ci
+
+# install cairo-rs-py
+git clone git@github.com:lambdaclass/cairo-rs-py.git
+pip install maturin
```

Comment:
Couldn't this be added as a poetry (dev) dependency?

Reply:
Done!